saleor/core/utils/random_data.py

```diff
@@ -481,13 +475,19 @@ def create_order_lines(order, discounts, how_many=10):
                 tax_rate=0,
             )
         )
-    Stock.objects.bulk_update(stocks, ["quantity", "quantity_allocated"])
     lines = OrderLine.objects.bulk_create(lines)
     manager = get_plugins_manager()
+    country = order.shipping_method.shipping_zone.countries[0]
     for line in lines:
         unit_price = manager.calculate_order_line_unit(line)
         line.unit_price = unit_price
         line.tax_rate = unit_price.tax / unit_price.net
+        warehouse = (
+            Warehouse.objects.filter(shipping_zones__countries__contains=country)
+            .order_by("?")
+            .first()
```

Comment:
I think we should move this Warehouse query outside the loop and add some counter to getting another warehouse for every line.